dist/tools/tunslip/tunslip6.c

```diff
@@ -650,9 +650,10 @@ devopen(const char *dev, int flags)
 {
     char t[1024];
     strcpy(t, "/dev/");
-    /* cppcheck-suppress bufferAccessOutOfBounds
-     * reason: seems to be a cppcheck bug */
-    strncat(t, dev, sizeof(t) - 5);
+    /* need to consider `\0` terminator and strlen("dev") = 5, hence 6
+     * cppcheck-suppress bufferAccessOutOfBounds
+     * (reason: seems to be a bug in cppcheck 1.7x) */
+    strncat(t, dev, sizeof(t) - 6);
```

Comment:
I think the comment on top is completely wrong. Anyway, this should rather go in a separate PR so it can be independently tested.

Reply:
just because it works, the comment is wrong? 

Looking at http://www.cplusplus.com/reference/cstring/strncat/ it states that the destination must be large enough to include both strings *plus* the `\0` terminator, so the calculation in the comment is correct, or not. One could rewrite this to `sizeof(t) - (strlen(dev) + 1)` to make it more obvious, but why bother to run `strlen(dev)` when we know its fixed to 5.

Reply:
Please let's discuss this in a separate PR. Regardless of right or wrong, this change does not belong here.